test/testpy/__init__.py

```diff
@@ -64,7 +64,10 @@ def GetCommand(self):
     source = open(self.file).read()
     flags_match = FLAGS_PATTERN.search(source)
     if flags_match:
-      result += flags_match.group(1).strip().split()
+      # PORT should match the definition in test/common.js.
+      env = { 'PORT': int(os.getenv('NODE_COMMON_PORT', '12346')) }
+      env['PORT'] += self.thread_id * 100
```

Comment:
Adding `thread_id * 100` to the port number and requiring the port to match `common.PORT` seem to be kind of contradictory?